tests/fast-integration/eventing-test/common/common.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+const eventMeshSecretFilePath = process.env.EVENTMESH_SECRET_FILE || '';
+const eventMeshNamespace = getEventMeshNamespace();
```

Comment:
(nit) Can we only call `getEventMeshNamespace()` if `eventMeshSecretFilePath` is not empty? Otherwise, we would have error log in console every time we run tests for nats.